typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.ts

```diff
@@ -251,13 +251,19 @@ export default class WellsLayer extends CompositeLayer<WellsLayerProps> {
 
             removeDuplicates(data);
 
-            const refine = this.props.refine;
-            data = refine
-                ? splineRefine(data) // smooth well paths.
-                : data;
-
             const coarseData = coarsenWells(data);
```

Comment:
Not sure it's a good idea to refine on a coarsened well trajectory.
Why did you change the algorithm order ?

Reply:
corasening is done first so that the refinements are done on the on the original data. coarsenWells returns a coaresened verion of the data but does not chenge the original data.

Reply:
OK, thank you